### PR TITLE
Change all foreman-release and pubkey.gpg downloads to HTTPS

### PR DESCRIPTION
**Pull request by domcleal**
Fixes GH-240

### CODE REVIEW

File: _includes/manuals/1.11/2.1_quickstart_installation.md [hunk 1]
```diff
@@ -147,7 +147,8 @@ yum -y install http://yum.theforeman.org/releases/{{page.version}}/f21/x86_64/fo
 {% highlight bash %}
 echo "deb http://deb.theforeman.org/ jessie {{page.version}}" > /etc/apt/sources.list.d/foreman.list
 echo "deb http://deb.theforeman.org/ plugins {{page.version}}" >> /etc/apt/sources.list.d/foreman.list
-wget -q http://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
+apt-get -y install ca-certificates
+wget -q https://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
```
Comment (GregSutcliffe) [inline _includes/manuals/1.11/2.1_quickstart_installation.md:151]: This won't work out of the box for Debian - https support isn;t enabled by default in apt (sigh, i know):

```
greg@topaz:~$ cat /etc/apt/sources.list.d/foreman.list 
deb https://deb.theforeman.org/ jessie 1.11
deb https://deb.theforeman.org/ plugins 1.11
greg@topaz:~$ sudo apt-get update
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
```

It's probably enough to add apt-transport-https to the preceeding line (and ditto all the ones below)

Comment (domcleal) [inline _includes/manuals/1.11/2.1_quickstart_installation.md:151]: The instructions on lines 148/149 remain HTTP, and it's only the wget (on this very line) that is HTTPS. It doesn't say to put an HTTPS URL into the sources file.

**Comment by GregSutcliffe** [comment]
Other than the Debian comment, any blockers to this now that https is enabled? It seems to work for me once I enabled apt-transport-https:

```
greg@topaz:~$ apt-cache policy foreman
 *** 1.11.2-1 0
        500 https://deb.theforeman.org/ jessie/1.11 amd64 Packages
```

File: _includes/manuals/1.11/2.1_quickstart_installation.md (hunk 1 above)
Comment (GregSutcliffe) [inline _includes/manuals/1.11/2.1_quickstart_installation.md:151]: Ah good point, I missed that. Let me rephrase then - do we want to recommend an https repo? I'm in two minds....

Comment (domcleal) [inline _includes/manuals/1.11/2.1_quickstart_installation.md:151]: I don't think so, I'm OK with leaving it HTTP and just ensuring that all public key downloads are HTTPS (also see #539). This also gives users the benefit of caching from any inline proxy servers.

Comment (GregSutcliffe) [inline _includes/manuals/1.11/2.1_quickstart_installation.md:151]: True enough, let's leave it be.

**Comment by domcleal** [comment]
> Other than the Debian comment, any blockers to this now that https is enabled?

Not that I'm aware of, it works OK in my tests.

**Comment by GregSutcliffe** [comment]
Merged as 85a4f36, ty @domcleal!
